geode-pulse/src/test/java/org/apache/geode/tools/pulse/tests/Server.java

```diff
@@ -99,105 +128,80 @@ public Server(int jmxPort, String properties, String jsonAuthFile) throws Except
       System.out.println("RMI registry ready.");
     } catch (Exception e) {
       System.out.println("Exception starting RMI registry:");
-      e.printStackTrace();
+      throw e;
     }
 
```

Comment:
No need to catch if you are just going to rethrow

Reply:
Good suggestion.